CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# 0.2.0
+
+* Add pagination functionality.
```

Comment:
Pagination is the mechanism that we've added support for larger clusters, but not the feature.

```suggestion
* Add support for clusters larger than 50 container instances.
```